packages/checkpoint/data_stream/firewall/elasticsearch/ingest_pipeline/default.yml

```
@@ -471,6 +471,10 @@ processors:
       field: checkpoint.ifdir
       target_field: network.direction
       ignore_missing: true
+  - convert:
+      field: checkpoint.bytes
+      type: long
+      ignore_missing: true
   - rename:
       field: checkpoint.bytes
       target_field: network.bytes
```

Thinking of doing this in a single step - 
```suggestion
  - convert:
      field: checkpoint.bytes
      target_field: network.bytes
      type: long
      ignore_missing: true
```

---

That leaves the value in `checkpoint.bytes`.